src/SFA.DAS.EmployerAccounts.Web/Models/CreateAccountModel.cs

```diff
@@ -18,5 +18,6 @@ public class CreateAccountViewModel
         public string OrganisationStatus { get; set; }
         public string EmployerRefName { get; set; }
         public string Sector { get; set; }
+        public HashedAccountIdModel HashedAccountId { get; set; }
```

Comment:
Is there a reason that this isn't just a string ?

Reply:
Err, just we already had a value type so kept it I think really, rather than go with primitive.

In this case the value type is merely a wrapper for a string  (no validation at construction etc. ), so wouldn't make much difference.

There could be an argument (in a re-factor) for this to not even be part of this model and the hashedaccountid storage retrieval to be 'behind' the Orchestrator.